src/test/java/edu/harvard/iq/dataverse/engine/command/impl/ReturnDatasetToAuthorCommandTest.java

```diff
@@ -0,0 +1,237 @@
+                        // We only need permissions for notifications, which we are testing in InReviewWorkflowIT.
+                        return Collections.emptyList();
+                    }
+                };
+            }
+
+        }
+        );
+    }
+
+
+    /*
+            if (theDataset == null) {
+            throw new IllegalCommandException("Can't return to author. Dataset is null.", this);
+        } 
+                
+        if (!theDataset.getLatestVersion().isInReview()) {
+            throw new IllegalCommandException("Latest version of dataset " + theDataset.getIdentifier() + " is not In Review. Only such versions my be returned to author.", this);
+        } 
+        
+        if (theDataset.getLatestVersion().getReturnReason() == null || theDataset.getLatestVersion().getReturnReason().isEmpty()){
+            throw new IllegalCommandException("You must enter a reason for returning a dataset to its author.", this);
+        }
+     */
+    @Test
+    public void testDatasetNull() {
+        dataset = null;
+        String expected = "Cannot return the dataset to the author(s) because it is null.";
+        String actual = null;
+        Dataset updatedDataset = null;
+
+        try {
+            updatedDataset = testEngine.submit(new ReturnDatasetToAuthorCommand(dataverseRequest, dataset, ""));
+        } catch (CommandException ex) {
+            actual = ex.getMessage();
+        }
+        assertEquals(expected, actual);
+        assertNull(updatedDataset);
+    }
+
+    @Test
+    public void testReleasedDataset() {
+        dataset.getLatestVersion().setVersionState(DatasetVersion.VersionState.RELEASED);
+        dataset.getLatestVersion().setInReview(true);
+        String expected = "This dataset cannot be return to the author(s) because the latest version is not In Review. The author(s) needs to click Submit for Review first.";
+        String actual = null;
+        Dataset updatedDataset = null;
+        try {
+            updatedDataset = testEngine.submit(new ReturnDatasetToAuthorCommand(dataverseRequest, dataset, ""));
+        } catch (CommandException ex) {
+            actual = ex.getMessage();
+        }
+        assertEquals(expected, actual);
+
+    }
+
+    @Test
+    public void testNotInReviewDataset() {
+        dataset.getLatestVersion().setVersionState(DatasetVersion.VersionState.DRAFT);
+        dataset.getLatestVersion().setInReview(false);
+        String expected = "This dataset cannot be return to the author(s) because the latest version is not In Review. The author(s) needs to click Submit for Review first.";
+        String actual = null;
+        Dataset updatedDataset = null;
+        try {
+            updatedDataset = testEngine.submit(new ReturnDatasetToAuthorCommand(dataverseRequest, dataset, ""));
+        } catch (CommandException ex) {
+            actual = ex.getMessage();
+        }
+        assertEquals(expected, actual);
```

Comment:
Your way might be a little cleaner, not sure if it's worth changing. I like the expected/actual pattern.